docs/docs/setting-up/data-ingestion/from-url.md

```diff
@@ -7,86 +7,87 @@ sidebar_position: 1
 To upload a file from a URL we will use the `bacalhau docker run` command.
 
 ```bash
-%%bash --out job_id
 bacalhau docker run \
     --id-only \
     --wait \
     --input https://raw.githubusercontent.com/filecoin-project/bacalhau/main/README.md \
     ghcr.io/bacalhau-project/examples/upload:v1
 ```
 
-The job has been submitted and Bacalhau has printed out the related job id. We store that in an environment variable so that we can reuse it later on.
+The job has been submitted and Bacalhau has printed out the related job id.
 
 ### Structure of the command
 
 Let's look closely at the command above:
 
-* `bacalhau docker run`: call to bacalhau 
+* `bacalhau docker run`: call to bacalhau using docker executor
 
-* `ghcr.io/bacalhau-project/examples/upload:v1`: the name and the tag of the docker image we are using
+* `ghcr.io/bacalhau-project/examples/upload:v1`: the name and version of the docker image we are using
 
-* ` --input=https://raw.githubusercontent.com/filecoin-project/bacalhau/main/README.md \`: URL path of the input data volumes downloaded from a URL source. 
+* `--input https://raw.githubusercontent.com/filecoin-project/bacalhau/main/README.md \`: URL path of the input data volumes downloaded from a URL source. 
```

Comment:
The \ here is just to escape the newline when typing multi-line commands in a terminal, we can probably remove that here.

Should this line also be swapped with the line above so the order of the explanation matches the order of the parameters in the example above?